man/adoc/bpftrace.adoc

```diff
@@ -2451,6 +2466,33 @@ BEGIN {
 }
 ----
 
+[#functions-getopt]
+=== getopt
+
+.variants
+* `int getopt(string arg_name)`
```

Comment:
Consider just requiring a default value?

Reply:
I like that we provide a terse option but I won't die on this hill. Though maybe I should change this behavior to not assume a boolean flag but rather we can just try to deduce the type from semantic analysis and if it's ambiguous we throw an error. This leaves the door open for a named argument `required` e.g. `getopt("hello", required=true)` that doesn't force a default value when the option is required.